hw/ip/pwrmgr/data/pwrmgr_testplan.hjson

```diff
@@ -132,46 +131,26 @@
       desc: '''
             Test random reset and reset_en.
 
-            The different reset inputs can be disabled via bits in the
-            `reset_en` CSR. Resets can be triggered either in active or
-            low power state.
+            Conditional reset inputs can be disabled via bits in the `reset_en`
+            CSR, while escalation and main power are unconditional. Resets can
+            be triggered either in active or low power state.
 
             **Stimulus**:
             - Sets `reset_en` randomly.
             - Randomly choose whether to put the unit in low power mode.
-            - Set `rstreqs_i` inputs randomly, both in value and in time.
+            - Generate resets randomly in value and time:
+              - Conditionals via rstreqs_i,
+              - Main power glitch via rst_main_ni.
+              - Escalation via `esc_rst_tx_i`.
 
             **Checks**:
             - Waits for fast fsm to become active reading `ctrl_cfg_regwen`
               until it becomes 1.
             - Checks the `reset_status` CSRs.
             - Checks `ip_clk_en` output has a low transition.
-            - Checks the output `pwr_rst_req.reset_cause` matches a reset
-              cause.
-            - Checks the output `pwr_rst_req.rstreqs` matches the enabled
-              resets.
-            '''
-      milestone: V2
-      tests: []
-    }
-    {
-      name: escalation_reset
```

Comment:
The testplan needs to account for all features discreetly. Our test strategy enables us to write on test to verify 3 seemingly different features together, however it is not always clear to the reader (for example, a SW person helping us at the chip level) if a very specific feature, for example, reset due to escalation is already tested at the pwrmgr IP level. The first place they will look is the regression results web page, which will only list `reset` as the testpoint. 

I suggest keeping this (and the main power glitch) as separate testpoints, but have them point the the same implemented test with the `tests` key. It should probably also be ok to have a reduced description for those:
```
    {
      name: escalation_reset
      desc: '''Test the escalation reset.

            This is covered by the `reset` testpoint above. Cross-referenced for 
            completeness. 
            '''
      milestone: V2
      tests: []
    }
```

Reply:
Good point. I will send another PR straightening this up.